### PR TITLE
DiegoFernandes.jsdesign version 1.0.2

### PR DESCRIPTION
**Pull request by oxygen-dioxide**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/38264)

### CODE REVIEW

File: manifests/d/DiegoFernandes/jsdesign/1.0.2/DiegoFernandes.jsdesign.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: DiegoFernandes.jsdesign
+PackageVersion: 1.0.2
+PackageLocale: en-US
+Publisher: Diego Fernandes
+PublisherUrl: https://js.design/
+PackageName: jsdesign
```
Comment (OfficialEsco) [inline manifests/d/DiegoFernandes/jsdesign/1.0.2/DiegoFernandes.jsdesign.locale.en-US.yaml:9]:
```suggestion
PackageName: 即时设计
```

File: manifests/d/DiegoFernandes/jsdesign/1.0.2/DiegoFernandes.jsdesign.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+PackageIdentifier: DiegoFernandes.jsdesign
+PackageVersion: 1.0.2
+PackageLocale: en-US
+Publisher: Diego Fernandes
+PublisherUrl: https://js.design/
+PackageName: jsdesign
+PackageUrl: https://js.design/
+License: Copyright © 2021 Diego Fernandes
+LicenseUrl: https://js.design/terms
+ShortDescription: ui design tool based on electron
```
Comment (OfficialEsco) [inline manifests/d/DiegoFernandes/jsdesign/1.0.2/DiegoFernandes.jsdesign.locale.en-US.yaml:13]:
```suggestion
ShortDescription: ui design tool based on electron
Moniker: jsdesign
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by OfficialEsco** [comment]
It appears you added the Moniker field twice 3 updates 2 suggested changes), i guess you clicked commit changes twice? O.o

File: manifests/d/DiegoFernandes/jsdesign/1.0.2/DiegoFernandes.jsdesign.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+PackageLocale: en-US
+Publisher: Diego Fernandes
+PublisherUrl: https://js.design/
+PackageName: 即时设计
+PackageUrl: https://js.design/
+License: Copyright © 2021 Diego Fernandes
+LicenseUrl: https://js.design/terms
+ShortDescription: ui design tool based on electron
+Moniker: jsdesign
+Moniker: jsdesign
```
Comment (ItzLevvie) [inline manifests/d/DiegoFernandes/jsdesign/1.0.2/DiegoFernandes.jsdesign.locale.en-US.yaml:15]:
```suggestion
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**